cloudify_cli/tests/resources/blueprints/local/windows_installers_blueprint.yaml

```diff
@@ -0,0 +1,30 @@
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/resources/rest-service/cloudify/types/types.yaml
```

Comment:
I agree with Ran. Generally, tests that don't actually need types.yaml, better not import it. This is mainly because these are unit tests after all and they should only reach S3 (outside world) if absolutely necessary.